src/EnergyPlus/UnitarySystem.cc

```diff
@@ -15045,32 +15028,22 @@ namespace UnitarySystems {
                                                                     CompName,
                                                                     FirstHVACIteration,
                                                                     PartLoadFrac,
-                                                                    CompIndex,
+                                                                    this->m_SuppHeatCoilIndex,
                                                                     QCoilActual,
                                                                     SuppHeatingCoilFlag,
-                                                                    FanOpMode,
+                                                                    this->m_FanOpMode,
                                                                     PartLoadFrac); // QCoilReq=PartLoadFrac 0.0 for this call
-                        if (!(this->m_SuppHeatCoilType_Num == DataHVACGlobals::Coil_HeatingDesuperheater) &&
-                            (this->m_DesignSuppHeatingCapacity > 0)) {
-                            PartLoadFrac = QCoilActual / this->m_DesignSuppHeatingCapacity;
-                        } else {
-                            if (QCoilActual > DataHVACGlobals::SmallLoad) {
-                                PartLoadFrac = 1.0;
-                            } else {
-                                PartLoadFrac = 0.0;
-                            }
-                        }
                     } break;
                     case DataHVACGlobals::Coil_HeatingElectric_MultiStage: {
-                        SpeedRatio = 0.0;
+                        // SpeedRatio = 0.0;
```

Comment:
Also set as declaration above, don't need to set again. Left here as comment so others know intent of calculaiton.